htdocs/accountancy/admin/index.php

```diff
@@ -499,6 +514,19 @@
 }
 print '</tr>';
 
+print '<tr class="oddeven">';
```

Comment:
I think this part of setup must be visible only if leterring is on ? If lettering is off, autolettering option has non sense, right ?

Reply:
You're right @eldy, I push a condition to see option